salt/utils/json.py

```diff
@@ -31,6 +31,44 @@ def find_json(raw):
     """
     ret = {}
     lines = __split(raw)
+    lengths = list(map(len, lines))
```

Comment:
The docstring needs to be updated if it is going to also ignore garbage at the end of the json.